pkg/frontend/metrics.go

```diff
@@ -0,0 +1,35 @@
+package frontend
+
+import (
+	"context"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/frontend/middleware"
+	"github.com/davecgh/go-spew/spew"
+)
+
+func (f *frontend) emitFrontendMetric(ctx context.Context, err error) {
```

Comment:
can this go into the middleware? (after the call to `h.ServeHTTP(w, r)`)

Reply:
How would you get the Internal Error code or Response code? If we would track the only Duration, this would work.